src/vaadin-overlay.html

```diff
@@ -322,7 +322,8 @@
           this._mouseUpInside = false;
           return;
         }
-        if (!this._last) {
+
+        if (!this._last || event.composedPath().filter(el => el.localName === 'vaadin-notification-card')[0]) {
```

Comment:
bad idea, `vaadin-overlay` should not know anything about components which are using it.